crates/context_aware_config/src/api/config/handlers.rs

```diff
@@ -576,9 +580,10 @@ async fn get(
 async fn get_resolved_config(
     req: HttpRequest,
     db_conn: DbConnection,
+    query_map: DynamicQuery<QueryMap>,
 ) -> superposition::Result<HttpResponse> {
     let DbConnection(mut conn) = db_conn;
-    let mut query_params_map = get_query_params_map(req.query_string())?;
+    let mut query_params_map = query_map.into_inner().into_inner();
```

Comment:
Looking at this code, I think we're doing something wrong. We're wrapping something twice and now have to call two accessors to get it? We should simplify. If we add Deref on QueryMap you can call the hashmap functions directly without needing the into_inner